lib/widgets/description.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+import 'package:movie_db_app/widgets/chiplist.dart';
+import '../shared/constants.dart';
+import 'package:movie_db_app/widgets/icon.dart';
+
+class Description extends StatelessWidget {
+  static const double dsOverviewFontSize = 17.0;
+  static const double dsSpaceBetweenChipList = 2;
+  static const double dsPadding = 32;
+  static const double dsSizedBoxHeight = 10;
+  static const double dsVoteAvgFontSize = 15.0;
+  static const double dsChipListFontSize = 10.0;
+
+  String description = '';
+  String voteAverage = '';
+  List<String> genres;
+
+  Description(
+      {super.key,
+      required this.description,
+      required this.voteAverage,
+      required this.genres,});
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.all(dsPadding),
+      child: Column(
+        children: [
+          Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              const DescriptionIcon(),
+              const SizedBox(height: dsSizedBoxHeight),
+              Text(
+                voteAverage,
+                style: const TextStyle(
+                  fontFamily: ConstantsClass.textFont,
+                  color: Colors.white,
+                  fontSize: dsVoteAvgFontSize,
+                ),
+              ),
+            ],
+          ),
+          const SizedBox(
+            height: dsSizedBoxHeight,
+          ),
+          Genres(
+            genres: genres,
+          ),
+          //In the future the idea is that if the user click on one genre, it redirects to another page where there are all the movies related to that genre
+          const SizedBox(
+            height: dsSizedBoxHeight,
+          ),
+          Text(
+            textAlign: TextAlign.justify,
+            style: const TextStyle(
+              fontFamily: ConstantsClass.textFont,
+              fontSize: dsOverviewFontSize,
+              color: Colors.white,
+            ),
```

Comment:
I'm seeing another text with the same style above, abstract the widget into a new file with this style as default

Reply:
solved